### PR TITLE
ECO-2285 Update step-engine version

### PR DESCRIPTION
**Pull request by profuel**
- Developer(s): @profuel 

- Ticket: https://spryker.atlassian.net/browse/ECO-2285

- Academy PR: ACADEMY_URL_HERE

- Release Group: URL_HERE

#### Please confirm

- [ ] No new OS components - or they have been approved by the legal department

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   Heidelpay             | patch                 |                       |

#### Release Notes

* Updated spryker/step-engine version
-----------------------------------------

##### Change log

Bugfixes

* Updated spryker/step-engine version

### CODE REVIEW

File: src/SprykerEco/Yves/Heidelpay/Form/AbstractHeidelpaySubForm.php [hunk 1]
```diff
@@ -63,4 +64,12 @@ public function getTemplatePath(): string
     {
         return static::PAYMENT_PROVIDER . '/' . static::PAYMENT_METHOD_TEMPLATE_PATH;
     }
+
+    /**
+     * @return string
+     */
+    public function getProviderName()
```
Comment (dereuromark) [inline src/SprykerEco/Yves/Heidelpay/Form/AbstractHeidelpaySubForm.php]: Are we using strict (return)typehinting yet in eco? Then you can use `: string` here

**Comment by dereuromark** [comment]
Please remove things like Updated README.md as those provide no value for changelog.
Only functional relevant things should be noted here.

File: src/SprykerEco/Yves/Heidelpay/Form/AbstractHeidelpaySubForm.php (hunk 1 above)
Comment (alex-galych) [reply src/SprykerEco/Yves/Heidelpay/Form/AbstractHeidelpaySubForm.php]: Done, thanks for the hint.

Comment (alex-galych) [reply src/SprykerEco/Yves/Heidelpay/Form/AbstractHeidelpaySubForm.php]: Ok

**Pull request merged**